Src/AutoFixtureUnitTest/DataAnnotations/StringLengthAttributeRelayTest.cs

```diff
@@ -112,5 +112,29 @@ public void CreateWithConstrainedStringRequestReturnsCorrectResult(int maximum)
             Assert.Equal(expectedResult, result);
             // Teardown
         }
+
+        [Theory]
+        [InlineData(1)]
+        [InlineData(2)]
+        [InlineData(3)]
+        public void CreateWithStringRequestConstrainedbyMinimunLengthReturnsCorrectResult(int maximum)

```

Comment:
This test is a *slightly* extended version of the [previous one](https://github.com/sergeyshushlyapin/AutoFixture/blob/4d526237d012bbfea42fe9e4e9d75db92d8dabc9/Src/AutoFixtureUnitTest/DataAnnotations/StringLengthAttributeRelayTest.cs#L94) which has got `MinimumLength` configured explicitly ...

```csharp
 var stringLengthAttribute = new StringLengthAttribute(maximum) { MinimumLength = 1 };
```

... instead of the default value used in previous test:

```
 var stringLengthAttribute = new StringLengthAttribute(maximum);
```

It's impossible to break the old test without breaking the new one. In the same time, the new one checks the scenario that is not covered in the old test. This makes the old test redundant, and if we all agree on that, I'd like to remove it.

Any objections?